arch/x86/kernel/cpu/resctrl/ctrlmondata.c

```diff
@@ -548,6 +548,9 @@ void mon_event_read(struct rmid_read *rr, struct rdt_resource *r,
 	rr->d = d;
 	rr->val = 0;
 	rr->first = first;
+	rr->arch_mon_ctx = resctrl_arch_mon_ctx_alloc(r, evtid);
+	if (IS_ERR(rr->arch_mon_ctx))
```

Comment:
missing rr->err = -EINVAL; (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=e557999f80a5ee4ec812f594ab42bb76c3ec4eb2)

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev